urbanopt/geojson-modelica-translator
# Update Packaging

#### Any background context you want to provide?
The dependencies were not resolving when using GMT as a dependent project. 

#### What does this PR accomplish?
* Add teaser and modelica builder as git checkouts to the setup.cfg `install_requires`
* Add JSONpath-ng

**Note: user will be required to install `requests` manually first before being able to install GMT. Run `pip install requests==2.22.0`**

#### How should this be manually tested?
* Easiest to verify that the package install correctly by running pip install in the geojson-modelica-translator-examples project.

#### What are the relevant tickets?
This ticket is already closed, but this commit belongs to that effort - #100 

#### Screenshots (if appropriate)
